### ISSUE TITLE
ImageNet pretrained model

### ISSUE THREAD

**Issue by frk1993** [RockyXu66/Faster_RCNN_for_Open_Images_Dataset_Keras#1]
Hi,
I found your Project very interesting. I wanted to ask if I can also pre initialize the Network with ImageNet pretrained weights?

Thank You!
Best Regards
FRK

**Comment by RockyXu66** [comment]
Hi FRK,
Of course, you can do that. It's kind of like transferring learning for our Faster R-CNN model.
 
Best regards,
Rocky

**Comment by RockyXu66** [comment]
What kinds of base model do you want to use? If you want to try other models like ResNet-50, you can follow the [original code](https://github.com/yhenon/keras-frcnn). If you want to use other CNN models, you need to build the structure of that model and load its corresponding weights.

**Comment by frk1993** [comment]
I want to use exactly vgg16 model as you have created (nn_base), but I want to initialize this nn_base with weights from ImageNet pre-trained VGG16 Network.
I hope I could explain it :)

**Comment by frk1993** [comment]
I think know I understand how you initialize the network. You initialize the network with weights from vgg16_weights_tf_dim_ordering_tf_kernels.h5 
Are this one ImageNet pre-trained weights ?

**Comment by RockyXu66** [comment]
Yes, this is the Keras pre-trained model which uses ImageNet as training dataset.
